crates/sui-graphql-rpc/src/types/owner.rs

```diff
@@ -28,6 +28,9 @@ use sui_types::gas_coin::GAS;
 #[derive(Clone, Debug)]
 pub(crate) struct Owner {
     pub address: SuiAddress,
+    /// If set, this is the version of the object designated as an Owner. Used to query the dynamic
+    /// fields of the object.
+    pub version: Option<u64>,
```

Comment:
I don't think it makes sense to set this on the `Owner` field -- the idea was to add an optional `parentVersion` parameter to `dynamic_field`, `dynamic_object_field`, and `dynamic_fields`.